activities/Abecedarium.activity/js/activity.js

```diff
@@ -35,6 +35,44 @@ define(["sugar-web/activity/activity","sugar-web/datastore"], function (activity
 		document.getElementById("stop-button").addEventListener('click', function (event) {
 			Abcd.sound.pause();
 		});
-	});
 
+		//Buttons to show and hide Language and Case Button
+		function hideLangandCasebutton(){
```

Comment:
It's never a good hide to repeat the same code in multiple files. I suggest to add these methods in util.js and prefix their name by Util.

Reply:
Ok Sir. I will keep that in mind.